### ISSUE TITLE
Problem with stageout S3 bucket when workspace integration is enabled

### ISSUE THREAD

**Issue by adwak** [EOEPCA/proc-ades#36]
When ADES is configured with workspace integration - the stageout S3 bucket is prepended with the 'projectid' which result in invalid bucket name.

```
[node-stage-out-pod-sovaocnf] Trying to init a root catalog catalog on s3://fake-project-id:dev-test/wf-c2a20f0c-c27d-11ed-9673-4a8fcfedd0ae/
[node-stage-out-pod-sovaocnf] The specified bucket is not valid.
[node-stage-out-pod-sovaocnf]    at Amazon.Runtime.Internal.HttpErrorResponseExceptionHandler.HandleExceptionStream(IRequestContext requestContext,ception, Stream responseStream)
[node-stage-out-pod-sovaocnf]    at Amazon.Runtime.Internal.HttpErrorResponseExceptionHandler.HandleExceptionAsync(IExecutionContext executionConte
[node-stage-out-pod-sovaocnf]    at Amazon.Runtime.Internal.ExceptionHandler`1.HandleAsync(IExecutionContext executionContext, Exception exception)
[node-stage-out-pod-sovaocnf]    at Amazon.Runtime.Internal.ErrorHandler.ProcessExceptionAsync(IExecutionContext executionContext, Exception except
[node-stage-out-pod-sovaocnf]    at Amazon.Runtime.Internal.ErrorHandler.InvokeAsync[T](IExecutionContext executionContext)
[node-stage-out-pod-sovaocnf]    at Amazon.Runtime.Internal.CallbackHandler.InvokeAsync[T](IExecutionContext executionContext)
[node-stage-out-pod-sovaocnf]    at Amazon.Runtime.Internal.EndpointDiscoveryHandler.InvokeAsync[T](IExecutionContext executionContext)
[node-stage-out-pod-sovaocnf]    at Amazon.Runtime.Internal.EndpointDiscoveryHandler.InvokeAsync[T](IExecutionContext executionContext)
[node-stage-out-pod-sovaocnf]    at Amazon.Runtime.Internal.CredentialsRetriever.InvokeAsync[T](IExecutionContext executionContext)
[node-stage-out-pod-sovaocnf]    at Amazon.Runtime.Internal.RetryHandler.InvokeAsync[T](IExecutionContext executionContext)
[node-stage-out-pod-sovaocnf]    at Amazon.Runtime.Internal.RetryHandler.InvokeAsync[T](IExecutionContext executionContext)
[node-stage-out-pod-sovaocnf]    at Amazon.Runtime.Internal.CallbackHandler.InvokeAsync[T](IExecutionContext executionContext)
[node-stage-out-pod-sovaocnf]    at Amazon.Runtime.Internal.CallbackHandler.InvokeAsync[T](IExecutionContext executionContext)
[node-stage-out-pod-sovaocnf]    at Amazon.S3.Internal.AmazonS3ExceptionHandler.InvokeAsync[T](IExecutionContext executionContext)
[node-stage-out-pod-sovaocnf]    at Amazon.Runtime.Internal.ErrorCallbackHandler.InvokeAsync[T](IExecutionContext executionContext)
[node-stage-out-pod-sovaocnf]    at Amazon.Runtime.Internal.MetricsHandler.InvokeAsync[T](IExecutionContext executionContext)
[node-stage-out-pod-sovaocnf]    at Amazon.S3.Transfer.Internal.SimpleUploadCommand.ExecuteAsync(CancellationToken cancellationToken)
....
```

**Comment by rconway** [comment]
Jira ticket EOEPCA-864 created for this - added to the current sprint #72.

[[#EOEPCA-864] [Ades] S3 URL with empty project ID.pdf](https://github.com/EOEPCA/proc-ades/files/10978507/EOEPCA-864.Ades.S3.URL.with.empty.project.ID.pdf)

**Comment by adwak** [comment]
Fixed in version [2.0.13](https://github.com/EOEPCA/proc-ades/releases/tag/2.0.13_5)
The new version also needs additional configuration options - [https://github.com/EOEPCA/eoepca/blob/develop/system/clusters/creodias/processing-and-chaining/proc-ades.yaml#L107](https://github.com/EOEPCA/eoepca/blob/develop/system/clusters/creodias/processing-and-chaining/proc-ades.yaml#L107)
